src/objectmap-jsonforms/package.json

```diff
@@ -42,6 +42,6 @@
   "types": "dist/index.d.ts",
   "dependencies": {
     "dot-object": "^2.1.4",
-    "session-data-mapper": "*"
+    "@fusebit/objectmap-utils": "^1.1.0"
```

Comment:
`"@fusebit/objectmap-utils": "*"` so it always uses the latest version when runing npm install

Reply:
do we want that? wouldn't we want it to be pinned to the version to avoid being broken?

Reply:
It's way more likely to break the package because you forgot to change the utils version to the latest one than making a breaking change on the utils library (it happened to me more times than i'd like to admit). Besides the latest version is always the one we want to use, because the utils component is specifically made to work with this component.